source/common/event/file_event_impl.cc

```diff
@@ -68,8 +69,10 @@ void FileEventImpl::assignEvents(uint32_t events) {
 }
 
 void FileEventImpl::setEnabled(uint32_t events) {
+  auto* base = event_get_base(&raw_event_);
+  ASSERT(base != nullptr);
```

Comment:
You ASSERT this in assignEvents() so I think you can remove this.

Reply:
Sharp eye!

Reply:
sharp eye!

Reply:
sharp eye!